docs/install.md

```diff
@@ -1,52 +1,113 @@
-
 ## Prerequisites
 
-### OCM based multi-cluster setup
-
-Ramen works as part of the [OCM hub] to orchestrate disaster-recovery, of
-application state, for applications deployed to [OCM managed clusters].
+### OCM managed multi-cluster setup
 
-As a result Ramen requires that multiple kubernetes clusters, that form the
-set of peer clusters, across which application deployment and state needs to
-be managed for disaster recovery are, managed clusters of an OCM hub.
+Ramen works as part of the [OCM hub](https://open-cluster-management.io/concepts/architecture/#hub-cluster)
+cluster to orchestrate the [placement](https://open-cluster-management.io/concepts/placement/)
+of [workloads](https://kubernetes.io/docs/concepts/workloads/) and their attachment
+to PersistentVolumes, on [OCM managed clusters](https://open-cluster-management.io/concepts/managedcluster/).
 
-The OCM hub orchestration is provided by Ramen using the
-[DRPlacementControl(DRPC) CRD](docs/drpc-crd.md).
+[Ramen hub](#ramen-hub) and [Ramen cluster](#ramen-cluster) components hence
+require an OCM managed multi-cluster setup for their operation.
 
-### Ceph-CSI as the volume provisioner
+### OCM Managed Cluster supporting VolumeReplication CRD
 
-Ramen also works as part of the [OCM managed clusters] to orchestrate,
+Ramen also works as part of the [OCM managed clusters](https://open-cluster-management.io/concepts/managedcluster/)
+to orchestrate,
 
 - [VolumeReplication](https://github.com/csi-addons/volume-replication-operator/blob/main/api/v1alpha1/volumereplication_types.go)
-  resources for all PVCs of an application
-- Preserving metadata regarding each PVC that is replicated
+  resources for all PVCs of a workload
+- Preserving relevant cluster data regarding each PVC that is replicated
 
 VolumeReplication resources require storage providers to support
 [CSI extensions](https://github.com/csi-addons/spec) that enable managing
 volume replication features for volumes provisioned by the storage provider.
-[ceph-CSI](https://github.com/ceph/ceph-csi/) is one such storage provider
+[Ceph-CSI](https://github.com/ceph/ceph-csi/) is one such storage provider
 that supports the required extensions.
 
-The managed cluster orchestration is provided by Ramen using the
-[VolumeReplicationGroup(VRG) CRD](docs/vrg-crd.md).
-
-**NOTE:** Ramen on OCM managed clusters and its orchestration of VRG CRD can
-be accomplished independent of an OCM setup, and only requires a native
-kubernetes cluster to manage VRG resources.
+[Ramen cluster](#ramen-cluster) component hence should be deployed to OCM
+managed clusters that support VolumeReplication extensions.
 
 ### S3 store
 
-Ramen preserves metadata related to VolumeReplication resources in an S3
-compatible object store. An S3 store endpoint is hence required as part of the
-setup.
+Ramen preserves cluster data related to PVC resources in an S3 compatible
+object store. An S3 store endpoint is hence required as part of the setup.
+
```

Comment:
I suggest stating the not so obvious (either here or elsewhere) that the PVC resource is user specified in the subscription's channel, is deployed by OCM to the managed clusters along with the application and thus need not be protected by Ramen.  Whereas, PV resource is dynamically provisioned in the managed cluster, is typically(?) not present in the channel and is hence, protected by Ramen to ensure correct binding of the PVC on the peer cluster with the appropriate replicated storage volume.

? This raises the question about applications that use statically provisioned PVs. We may want to clarify if and how we support it.

Reply:
The existing NOTE just below addresses this already?

Do we really need to talk about how Ramen works in here? That could be a different document for the curious I would think.

Currently PV cluster data protection and reestore is something that is imposed by VolumeReplication, so if a storage provider supports VolRep we are done.